examples/kubernetes/konnectivity-agent.yaml

```diff
@@ -20,6 +26,7 @@ spec:
       "--ca-cert=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt",
       "--proxy-server-host=${CLUSTER_IP}",
       "--proxy-server-port=8091",
+      "--service-account-token-path=/var/run/secrets/tokens/konnectivity-agent-token",
```

Comment:
afaik a pod can only be bound to one service account, and the service account cannot be changed after pod creation. I noticed that the token info is mounted to `var/run/secrets/kubernetes.io/serviceaccount/token` by default. I think we can directly reference that token instead of having to mount the volumes below?

cc: @caesarxuchao

Reply:
cc: @mikedanese